core/src/impl/Kokkos_Utilities.hpp

```diff
@@ -414,7 +415,12 @@ struct inclusive_scan_integer_sequence {
 };
 
 template <typename T>
-using identity_t = T;
+struct identity {
```

Comment:
Rename this to type_identity to match C++20

Reply:
we are doing that later.